internal/cli/atlas/security/ldap/status.go

```diff
@@ -49,13 +49,13 @@ func (opts *StatusOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mongocli atlas ldap status <ID> [--projectId projectId]
+// mongocli atlas ldap verify status <ID> [--projectId projectId]
 func StatusBuilder() *cobra.Command {
 	opts := &StatusOpts{}
 	cmd := &cobra.Command{
 		Use:   "status <ID>",
 		Args:  cobra.ExactValidArgs(1),
-		Short: verify,
+		Short: status,
```

Comment:
[q] Is `verify` removed, as I see there's still a definition in `description.go`?

Reply:
no, I forgot to change the description of the status command in the previous PR. The verify command is still there.